README.md

```diff
@@ -96,9 +96,11 @@ The `start` method start (and restarts) mathjax-node. This allows reconfiguratio
 
 **Note.** This is done automatically when `typeset` is first called (see below).
 
-### `typeset(options, callback)`
+### `typeset(options[, callback])`
 
-The `typeset` method is the main method of mathjax-node. It expects a configuration object `input` and a `callback`.
+The `typeset` method is the main method of mathjax-node. It expects a configuration object `input` and optionally a callback.
```

Comment:
Shouldn't the configuration object be called `options` rather than `input`?

Reply:
I didn't change it :smile:. I always think of it as input (since it contains the actual input). But I'm ok with changing it.

Reply:
I know you didn't change it, but I did notice that it is not the same as the name above.  You could change `options` to `input` in the `typeset()` call if you prefer (and then update `options` elsewhere as well).